test/e2e/login-helpers.js

```diff
@@ -88,6 +88,7 @@ function logIn(token, client, url, level) {
 
   client
     .url(`${E2eHelpers.baseUrl}${url}`)
+    .pause(500)
```

Comment:
Had to introduce a short pause here to allow the client to redirect. Not sure why it's necessary now.

Reply:
Actually would really not like to have this if possible, since it's indicative of brittle tests. Without it, the test will hang for a long time, much longer the timeout specified in the wait, and the redirect doesn't happen in the meantime. Introducing the pause somehow allows the `window.location.replace` to occur, but why?